### PR TITLE
fix(datepicker): stop event bubbling from buttons

### LINKED ISSUE #5347: Popover closes unexpectedly
Example:
http://plnkr.co/edit/dMWd7KRHjaoOZWXyYvE5?p=preview

1. When I click on "today" the popover closes unexpectedly.
2. When I select a date the calendar closes, again I select a new date in the same calendar and closes unexpectedly the popover. This happens when I select twice dates in the same calendar.

Reference: https://github.com/angular-ui/bootstrap/issues/5305#issuecomment-173965779

### PR DESCRIPTION
**Pull request by wesleycho**
- Stop event bubbling from clicks in popup

Fix in action [here](http://plnkr.co/edit/YAzEdlmz8St6PtzLGPDF?p=preview).

This fixes #5347

### CODE REVIEW

File: template/datepicker/popup.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
   <ul class="uib-datepicker-popup dropdown-menu" dropdown-nested ng-if="isOpen" ng-style="{top: position.top+'px', left: position.left+'px'}" ng-keydown="keydown($event)" ng-click="$event.stopPropagation()">
     <li ng-transclude></li>
     <li ng-if="showButtonBar" class="uib-button-bar">
-    <span class="btn-group pull-left">
-      <button type="button" class="btn btn-sm btn-info uib-datepicker-current" ng-click="select('today')" ng-disabled="isDisabled('today')">{{ getText('current') }}</button>
-      <button type="button" class="btn btn-sm btn-danger uib-clear" ng-click="select(null)">{{ getText('clear') }}</button>
-    </span>
-      <button type="button" class="btn btn-sm btn-success pull-right uib-close" ng-click="close()">{{ getText('close') }}</button>
+      <span class="btn-group pull-left">
```
Comment (icfantv) [inline template/datepicker/popup.html]: formatting tool went awry?

Comment (wesleycho) [inline template/datepicker/popup.html]: That was me fixing the indentation

Comment (icfantv) [inline template/datepicker/popup.html]: ooooooh.  ok.  i couldn't see it before.  sorry.

**Comment by Foxandxss** [comment]
LGTM

**Pull request closed without merging**